app/javascript/controllers/search_controller.js

```diff
@@ -0,0 +1,18 @@
+import { Controller } from "@hotwired/stimulus"
+import Rails from "@rails/ujs"
+
+export default class extends Controller {
+  handleSubmit(event) {
```

Comment:
Submits the search form to `SearchesController.create` and replaces the existing partial with the updated one returned.